projects/packages/search/src/dashboard/components/record-meter/index.jsx

```diff
@@ -13,6 +13,9 @@ import { useSelect } from '@wordpress/data';
  * Internal dependencies
  */
 import { STORE_ID } from 'store';
+import { BarChart } from './barchart.jsx';
```

Comment:
We don't need to include file extensions on these imports.

Reply:
ah, right - thank you! 👍